students/sunday-class.md

```diff
@@ -30,7 +30,7 @@ This is a co-learning moment for each one of you.
 
 Your coaches will spend some time leading a review session of last week's assignments.
 
-The best way for you to help your coaches give a helpful review session is to **create [Sunday Review Issues](https://github.com/aame1/home/blob/master/students/asking-for-help.md)** in your class repo.  This way they can prepare clear examples and presentations ahead of time.
+The best way for you to help your coaches give a helpful review session is to **create [Sunday Review Issues](https://github.com/HackYourFutureBelgium/home/blob/master/students/asking-for-help.md)** in your class repo.  This way they can prepare clear examples and presentations ahead of time.
```

Comment:
Good catch!  can you update your PR one more time?  Right now you have an absolute link to the source file which will open in a new tab.  If you change it to a relative link (`./asking-for-help.md`) then the link will stay in the tab and open a new page in the gitbook.

this mistake you found is from before the gitbook, when all the guides were in separate repos

Reply:
hi @colevandersWands 

committed the new relative link, but, i can only "view pull request" now. :) 
I'm still not very familiar with conflicts and these scenarios. :hatching_chick: